frontend/src/components/CopyToClipboard.tsx

```diff
@@ -13,7 +13,13 @@ function CopyToClipboard({ textToCopy }: CopyProps) {
 
   const handleCopy = () => {
     navigator.clipboard.writeText(textToCopy).then(
-      () => setCopied(true),
+      () => {
+        setCopied(true);
+        // Reset 'copied' state after 3 seconds
```

Comment:
This comment doesn't seem to add much.

Arguably the same for the comment below.

Reply:
Okey, I have removed the comment